packages/expo-local-authentication/android/src/main/java/expo/modules/localauthentication/LocalAuthenticationModule.java

```diff
@@ -87,15 +87,23 @@ public void supportedAuthenticationTypesAsync(final Promise promise) {
       promise.resolve(results);
       return;
     }
-    if (mPackageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT)) {
-      results.add(AUTHENTICATION_TYPE_FINGERPRINT);
-    }
-    if (mPackageManager.hasSystemFeature(PackageManager.FEATURE_FACE)) {
-      results.add(AUTHENTICATION_TYPE_FACIAL_RECOGNITION);
+
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+      if (mPackageManager.hasSystemFeature("android.hardware.fingerprint")) {
+        results.add(AUTHENTICATION_TYPE_FINGERPRINT);
+      }
     }
-    if (mPackageManager.hasSystemFeature(PackageManager.FEATURE_IRIS)) {
-      results.add(AUTHENTICATION_TYPE_IRIS);
+
+    if (Build.VERSION.SDK_INT >= 29) {
+      if (mPackageManager.hasSystemFeature("android.hardware.biometrics.face")) {
```

Comment:
Can you also put some comments to replace these strings with appropriate PackageManager properties in the future once we drop support for Android SDK 28?